ui/app/models/pki/certificate/base.js

```diff
@@ -33,7 +34,6 @@ export default class PkiCertificateBaseModel extends Model {
     assert('You must provide a helpUrl for OpenAPI', true);
   }
 
-  // Required input for all certificates
```

Comment:
though this is the default, this behavior can be changed when configuring a role